transcom/mymove
# 19782 shipment cancellation date validation be int quick fix

## [B-19782](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A946146)

## Summary

This PR prevents the TOO from cancelling any shipment that's 'actualPickupdate' is on or after the request date.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Make a HHG 
2. Login as SC to edit and submit move 
4. Login as TOO and  approve and send move
5. Go to the 'Move Task Order' tab and try to cancel the move
6. Confirm you're able to delete the move without an actualPickupdate

### Backend


## Screenshots
<img width="1049" alt="Screenshot 2024-06-07 at 10 23 00 AM" src="https://github.com/transcom/mymove/assets/146969726/07812744-591d-42aa-80aa-951c3b090281">
